packages/salesforcedx-sobjects/src/generator/fauxClassGenerator.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as fs from 'fs';
+import * as path from 'path';
+import { SObjectDescribe } from '../describe';
+
+export class FauxClassGenerator {
```

Comment:
This is also another contract point for faux class generation?

Reply:
Yes

Reply:
Yes